config/environments/production.rb

```diff
@@ -26,7 +26,7 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
```

Comment:
@hoangmirs Why are we doing this? In production we should let it be false I guess ?

Reply:
I'm trying to remove sprockets and using webpacker only.
I updated in https://github.com/nimblehq/nimble-survey-web/pull/20/commits/f0703b1fb380a0cb8ae57cbce0a678e694f8c3f6, I removed all the related code instead of comment them, follow [this guide](https://stackoverflow.com/a/59958831/6339862)